website/docs/podman/accessing-podman-from-another-wsl-instance.md

```diff
@@ -0,0 +1,207 @@
+---
+sidebar_position: 60
+title: Accessing Podman from another WSL distribution
+description: On Windows, access your Podman Desktop containers from another WSL distribution.
+---
+
+# Accessing Podman from another WSL distribution
+
+On Windows, Podman Desktop configures Podman in a dedicated Windows Subsystem for Linux (WSL) distribution: the Podman Machine.
+It also configures the Windows Podman client to communicate with the Podman Machine.
+But it does not configure your other WSL distributions.
+
+You might have other WSL distributions running, and want to access from there to your Podman Desktop containers.
+
+This tutorial focuses on the Ubuntu distribution of Linux, to walk you through the steps to configure your WSL distribution:
+
+1. Install the Podman client that will initiate requests to the remote Podman.
+2. Configure access to the Podman Machine defined by Podman Desktop.
```

Comment:
As a reader, it's critical to explain why you are asking me to do certain things, so I can understand what's behind and how it works.